packages/document/main-doc/docs/en/guides/advanced-features/compatibility.mdx

```diff
@@ -22,32 +22,11 @@ ios_saf >= 10
 Please refer to [Modern.js Builder - Browserslist](https://modernjs.dev/builder/en/guide/advanced/browserslist) for more information.
 :::
 
-## Browserslist
```

Comment:
This chapter is removed in CN docs , and refer to Builder docs , so I delete it.